k8s-deployer/scripts/publish-report.sh

```diff
@@ -34,6 +32,13 @@ then
   exit 1
 fi
 
+if [ "${USERNAME}" == "" ];
```

Comment:
I don't think this variable has anything to do with git. `zsh` sets `$USERNAME` env variable. And when you are in bash, the variable doesn't have any value

<img width="644" alt="image" src="https://github.com/kindredgroup/pit-toolkit/assets/118979108/1e8c493a-44b0-4035-b1e7-fea45c25ecee">

Reply:
It is also set by jenkins